kapitan/cli.py

```diff
@@ -462,29 +437,13 @@ def secret_update_validate(args, ref_controller):
         if 'secrets' not in kap_inv_params:
             raise KapitanError("parameters.kapitan.secrets not defined in {}".format(target_name))
 
-        try:
-            try:
-                recipients = kap_inv_params['secrets']['gpg']['recipients']
-            except KeyError:
-                # TODO: Keeping gpg recipients backwards-compatible until we make a breaking release
-                logger.warning("WARNING: parameters.kapitan.secrets.recipients is deprecated, " +
-                               "please use parameters.kapitan.secrets.gpg.recipients")
-                recipients = kap_inv_params['secrets']['recipients']
-        except KeyError:
-            recipients = None
-
-        try:
-            gkey = kap_inv_params['secrets']['gkms']['key']
-        except KeyError:
-            gkey = None
-
-        try:
-            awskey = kap_inv_params['secrets']['awskms']['key']
-        except KeyError:
-            awskey = None
-
         for token_path in token_paths:
             if token_path.startswith("?{gpg:"):
+                try:
+                    recipients = kap_inv_params['secrets']['gpg']['recipients']
```

Comment:
Do we want to redefine recipients for every token_path? I believe that's why it was before the for loop.

Reply:
Oops, good spot.